coding-concepts.js

```diff
@@ -5,13 +5,22 @@
 // First, predict the outcome based on your understanding of the code.
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
+
+
+
+
 // --------------------1) What will this log?
 
 var cohort = "Alpha 2022"
 // console.log(cohort.split(""))
 
 // a) Your answer:
```

Comment:
Appreciate the explanation. Please also include exactly how you expect the output to appear. For instance, based on your words. The solution would appear as   'A', 'l', 'p', 'h', 'a', ' ', '2', '0', '2', '2'. After you un-comment the console.log you would see ['A', 'l', 'p', 'h', 'a', ' ', '2', '0', '2', '2'] that it returns as an array with each character split including the space, then explain why.